hole/morse.go

```diff
@@ -55,16 +55,13 @@ func morse(reverse bool) []Scorecard {
 		string(shuffle([]byte("ABCDEFGHIJKLMNOPQRSTUVWXYZ"))),
 	})
 
-	args := []string{strings.Join(words, " ")}
+	text := strings.Join(words, " ")
 
-	if reverse {
-		args = []string{strings.TrimSpace(args[0][:36]), strings.TrimSpace(args[0][36:])}
-	}
-
-	scorecards := make([]Scorecard, len(args))
+	scorecards := make([]Scorecard, 2)
 
-	for i, arg := range args {
+	for i := 0; i < 2; i++ {
 		out := ""
+		arg := text[:36]
```

Comment:
Maybe I've misread something, but doesn't this need whitespace trimmed?

Reply:
Yes, thanks